CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Alfred API - Changelog
 
+## 5.0.2 (2024-05-13)
```

Comment:
Please change date to `(unreleased)`

(and I see now that the post-release commit was forgotten in the previous release. Thanks for adding it.)

Reply:
We need this release to push Vespa to production on ScaleWay

Reply:
I'm not following... Sure, you can release it. This is always a separate ticket that fill set the date here.